onnxruntime/core/framework/debug_node_inputs_outputs_utils.cc

```diff
@@ -231,19 +240,30 @@ const NodeDumpOptions& NodeDumpOptionsFromEnvironmentVariables() {
   return node_dump_options;
 }
 
+static bool AnyOptionIsSet(const NodeDumpOptions& dump_options) {
```

Comment:
if we have a NodeDumpOptions::DumpFlags::None value, we can directly check for equality

Reply:
as a follow up comment, is a separate function needed for `dump_options.dump_flags != NodeDumpOptions::DumpFlags::None`? if you'd like to put it in one, i think the current name is a bit too general